### ISSUE TITLE
Cannot play DSD audio

### ISSUE THREAD

**Issue by felix920506** [jellyfin-archive/jellyfin-desktop-qt#581]
**Describe the bug**
When trying to play DSD audio, JMP errors with "The server isn't sending a compatible format"

**To Reproduce**
1. Add DSD audio to Server
2. Play DSD audio
3. See Error

**Expected behavior**
Expected audio to play

**Desktop (please complete the following information):**
 - OS: MacOS 
 - Version 13.1
 - Installation Method: DMG downloaded from jellyfin.org
 
**Additional context**
It shows that it is direct playing but the server is trying to transcode for whatever reason, then fails
ffmpeg logs:
[Log.log](https://github.com/jellyfin/jellyfin-media-player/files/14625954/Log.log)

JMP logs:
[Jellyfin Media Player拷貝.log](https://github.com/jellyfin/jellyfin-media-player/files/14625953/Jellyfin.Media.Player.log)

**Comment by sevenrats** [comment]
can anything in jellyland play DSD audio?

**Comment by felix920506** [comment]
Web transcode plays it just fine

**Comment by sevenrats** [comment]
it may be a symptom of jmp's always-directplay-then-failover method

**Comment by retrography** [comment]
Bump. I am using Finamp as client. I don't get any error. The song just won't advance.

**Comment by felix920506** [comment]
@retrography If you are having problems with finamp you should report it in their repo

**Comment by felix920506** [comment]
> it may be a symptom of jmp's always-directplay-then-failover method

@sevenrats I don't think it's this. It does request a transcode, but because of how DSD works (not PCM), JMP grabs the "sample rate" of the source and requests a transcode to flac with sample rate of like 2.8Mhz (same as source) which won't work

**Comment by retrography** [comment]
@felix920506 You are right. The problem is Finamp. DSD playback works correctly on the web interface.

**Comment by andrewrabert** [comment]
This Qt desktop client has been deprecated in favor of a rewrite: https://github.com/jellyfin/jellyfin-desktop
